test/nodejs-bundle/circuit/helpers/utils.js

```diff
@@ -0,0 +1,110 @@
+'use strict'
```

Comment:
This file should be part of the utils at the top `test/utils` It is also duplicating code that is already there.